pkg/op-brains/op_brains/chat/classify.py

```diff
@@ -0,0 +1,19 @@
+from langchain_openai import ChatOpenAI
+from langchain_anthropic import ChatAnthropic
+from typing import List, Tuple
+from op_brains.summarizer.utils import Prompt, QuestionClassificationStructured
+from op_brains.chat.apis import access_APIs
+
+
+async def classify_question(question: str, thread: List = []) -> str:
```

Comment:
I thought the approach was to use the categories from the forum using the references used in the response. Can you explain more about why you ended up using this approach? This way we cant ever guarantee there's only going to be a limited set of categories right? How did you think we'd analyze this data?

Also, doesn't this add even more latency to the answer? Do you think the value added is worth it compared to the other approach?